classes/external/map_assessment.php

```diff
@@ -43,6 +43,7 @@ public static function execute_parameters() {
             'sourcetype' => new external_value(PARAM_TEXT, 'Source Type', VALUE_REQUIRED),
             'sourceid' => new external_value(PARAM_INT, 'Source ID', VALUE_REQUIRED),
             'mabid' => new external_value(PARAM_INT, 'Assessment Component ID', VALUE_REQUIRED),
+            'reassess' => new external_value(PARAM_INT, 'Reassessment Flag', VALUE_DEFAULT, 0),
```

Comment:
should we make it required?

Would have to be a conscious decision to be fetching one or the other & create an awareness that the reassess option exists